frontend/www/js/omegaup/arena/contest_contestant.ts

```diff
@@ -262,6 +322,27 @@ OmegaUp.on('ready', () => {
   });
   socket.connect();
 
+  function getSelectedValidTab(tab: string, isAdmin: boolean): string {
+    const validTabs = ['problems', 'ranking', 'runs', 'clarifications'];
+    const defaultTab = 'problems';
+    if (tab === 'runs' && !isAdmin) return defaultTab;
+    const isValidTab = validTabs.includes(tab);
+    return isValidTab ? tab : defaultTab;
+  }
+
+  if (payload.allRuns) {
+    for (const run of payload.allRuns) {
+      trackRun({ run });
+    }
+  }
+
+  if (locationHash[1] && locationHash[1].includes('show-run:')) {
+    const showRunRegex = /.*\/show-run:([a-fA-F0-9]+)/;
+    const showRunMatch = window.location.hash.match(showRunRegex);
+    contestContestant.guid = showRunMatch ? showRunMatch[1] : null;
```

Comment:
```suggestion
    contestContestant.guid = showRunMatch?.[1] ?? null;
```

El [Optional Chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining#array_item_access_with_optional_chaining) se puede usar con arreglos!

Reply:
Muy interesante. Listo el cambio.